clab/graph_templates/nextui/static/js/script.js

```diff
@@ -89,8 +89,31 @@
             'setModel': function (model) {
                 this.inherited(model);
             },
+            calculateLabelPosition: function(percentDistance) {
+              var l = this.line();
+              var n = this.line().normal().multiply(this.getOffset()*3);
```

Comment:
what happened here: 26,455 additions

Reply:
I also updated the next.js script to make it the non-minified version.  I think it will be easier to maintain in the future for the limited future it has left.  I had to update the next.js to use the positionAlongCurve method.  I'm happy to change back to the minified version if you want, but the framework no longer being worked on and seems easier for the next person.